src/map/addPopUp.js

```diff
@@ -2,175 +2,190 @@ import Mustache from 'mustache';
 import { getCurrentState } from '../store/actions/actions';
 import commaFormatting from './../utils/commaFormatting';
 
+/**
+ *  Adds popup to the map
+ * @param {''} mapId - Identified what map to show popup on
+ * @param {{}} mapboxGLMap mapbox map object allows us to interact with map methods from mapbox
+ * @param {() => {}} dispatch redux function that accepts an action 
```

Comment:
Should we use `string`, `object`, `function` for the param typings?

Reply:
you're right changing that